modin/distributed/dataframe/pandas/partitions.py

```diff
@@ -99,8 +106,13 @@ def get_block(partition):
 
 
 def from_partitions(
-    partitions, axis, index=None, columns=None, row_lengths=None, column_widths=None
-):
+    partitions: list,
+    axis: Optional[int],
+    index: Optional[list] = None,
+    columns: Optional[list] = None,
```

Comment:
This `list` doesn't feel correct, `index` and `columns` could easily be descendants of `pandas.Index` or `ndarray`, maybe we better specify them as `Optional[Sequence]` instead?

Reply:
Done.